### ISSUE TITLE
Google Drive Links

### ISSUE THREAD

**Issue by UltraTech66** [Nirvan66/geoguessrLSTM#2]
Hello, the Google Drive links are private.  Could you please make them public? 
Thanks.

**Comment by rackaracka123** [comment]
I agree with Mr Ultra Tech 66 on this one. I would love to try the AI for a spin :)
